app/serializers/location_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class LocationSerializer < ActiveModel::Serializer
+  attributes :id, :name, :city, :state, :price, :password_protected, :average_rating
```

Comment:
uhhhhh is there a reason that `password_protected` is here? Otherwise pls remove

Reply:
We are passing a "password_protected" status down meaning if a wifi requires a password or not